Readme.md

```diff
@@ -229,6 +229,16 @@ This tool has been developed to be able to test data channel implementations
 without having to write the required DTLS and ICE stacks. An example of such a
 testing tool is [dctt][dctt] which uses the kernel SCTP stack of FreeBSD.
 
+Building:
+
+This tool is not built by default. In order to build it, set the cached CMake flag `SCTP_REDIRECT_TRANSPORT` to `ON` when building, like so: 
```

Comment:
I'd replace *cached CMake flag* with *environment variable* as the internals don't really matter to the user.